packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dedicatedCloud.service.js

```diff
@@ -536,6 +536,47 @@ class DedicatedCloudService {
     });
   }
 
+  /* ------- IAM -------*/
+
+  getIamStatus(serviceName) {
+    return this.$http
+      .get(`/dedicatedCloud/${serviceName}/iam`)
+      .then(({ data }) => data);
+  }
+
+  isIamTogglable(serviceName, state, _returnAsBoolean = true) {
```

Comment:
**nitpick**: in every call to this function :
- `_returnAsBoolean` is set to `true`
- The return value of this method is used like this `response.toggable`

Therefore, please remove the `_returnAsBoolean` argument and make the method just return a simple boolean instead of an object

Reply:
ok, will change that. I though it was a convention to do it like that by reading other parts of the code.